cloudigrade/api/tests/test_helper.py

```diff
@@ -36,9 +36,9 @@ class SandboxedRestClientTest(TestCase):
 
     def setUp(self):
         """Set up user for the client tests."""
-        self.username = _faker.random_int(min=100000, max=999999)
+        self.account_number = _faker.random_int(min=100000, max=999999)
```

Comment:
I think we should go ahead and add a random `org_id` and use it in the `generate_test_user` call below so we don't forget later when it becomes the primary identifier.

Reply:
👍